cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+  "baseUrl": "http://localhost:3000"
```

Comment:
This is handy to have, as Cypress will warn that the server isn't available (instead of waiting for tests to fail or timeout).